HW9/package3/Main.java

```diff
@@ -0,0 +1,61 @@
+package package3;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        List<Integer> listInt = fillIntList();
+
+        System.out.println("list: ");
+        for (int i : listInt) System.out.print(i + " ");
+        System.out.println();
+
+        findUnique(listInt);
+        findUniqueFor(listInt);
+    }
+
+    static List<Integer> fillIntList() {
+        int listLength = (int) (Math.random() * 90) + 10;
+        List<Integer> list = new ArrayList<>(listLength);
+        for (int i = 0; i < listLength; i++) {
+            list.add((int) (Math.random() * 90) + 10);
+        }
+        return list;
+    }
+
+    static void findUnique(List<Integer> list) {
+        Set<Integer> set = new HashSet<>(list);
+        list.clear();
```

Comment:
This is a bad idea to reuse n object (in your case list) - "clean code" dictates - one variable - one logic\use. So, if `list` is a method parameter that holds "not unique" objects and you have to create a list of unique objects - create a separate variable (`List`) for it.